servers/aero.js

```diff
@@ -0,0 +1,29 @@
+global.Promise = require('bluebird');
```

Comment:
Fair enough. I used it because the author uses it in his own benchmark here. You're right, probably better to remove it.